### PR TITLE
add --utc like journalctl

### PR DESCRIPTION
**Pull request by lionello**

### CODE REVIEW

File: src/cmd/cli/command/commands.go [hunk 1]
```diff
@@ -614,14 +615,23 @@ var tailCmd = &cobra.Command{
 		var etag, _ = cmd.Flags().GetString("etag")
 		var raw, _ = cmd.Flags().GetBool("raw")
 		var since, _ = cmd.Flags().GetString("since")
+		var utc, _ = cmd.Flags().GetBool("utc")
 
-		ts, err := cli.ParseTimeOrDuration(since)
+		if utc {
+			os.Setenv("TZ", "UTC")
```
Comment (nullfunc) [inline src/cmd/cli/command/commands.go:621]: I don't see that we are referencing the env "TZ" anywhere.

**Review by nullfunc — COMMENTED** [review]
Had a couple of comments/questions

File: src/pkg/cli/tail.go [hunk 2]
```diff
@@ -46,7 +46,10 @@ type P = client.Property // shorthand for tracking properties
 
 // ParseTimeOrDuration parses a time string or duration string (e.g. 1h30m) and returns a time.Time.
 // At a minimum, this function supports RFC3339Nano, Go durations, and our own TimestampFormat (local).
-func ParseTimeOrDuration(str string) (time.Time, error) {
+func ParseTimeOrDuration(str string, now time.Time) (time.Time, error) {
+	if str == "" {
+		return time.Time{}, nil
```
Comment (edwardrf) [inline src/pkg/cli/tail.go:51]: Maybe we should be more explicit about the default time value, or add comment explaining the zero time means jan 1 year 0001.

**Review by edwardrf — APPROVED** [review]
LGTM

File: src/cmd/cli/command/commands.go (hunk 1 above)
Comment (lionello) [reply src/cmd/cli/command/commands.go:621]: Golang does that

**Pull request merged**